.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
```

Comment:
Why single quotes? (just curious)

Reply:
Its seems to be quite common. Just copied it.